public/app/features/explore/ContentOutline/ContentOutlineItem.tsx

```diff
@@ -2,27 +2,54 @@ import React, { useEffect, useRef, ReactNode } from 'react';
 
 import { useContentOutlineContext } from './ContentOutlineContext';
 
+type INDENT_LEVELS = 'root' | 'child';
+
 export interface ContentOutlineItemBaseProps {
+  panelId: string;
   title: string;
   icon: string;
+  customTopOffset?: number;
+  level?: INDENT_LEVELS;
+  mergeSingleChild?: boolean;
```

Comment:
nit: could we have a comment on what this is for?

Reply:
done :)